scripts/travis/travis-tests

```diff
@@ -59,6 +73,10 @@ travis_prep () {
 	apt-get update -qq
 	apt-get install -qq --no-install-recommends $TRAVIS_PKGS
 	chmod a+x $HOME
+
+	# zdtm uses an unversioned python binary to run the tests.
+	# let's point python to python3
+	ln -sf /usr/bin/python3 /usr/bin/python
```

Comment:
I don't like this hack. Why we can't use the default python version for each distribution?

Reply:
Not every distribution sets a python link. When there was only python2, python was a link to python2. When python3 was released there was a PEP that there should be not python link to python3. But that was later changed. I think. So there are some distributions which are providing a python to python3 link and some do not provide such a link.

As we are using an unversioned python in zdtm we need an unversioned python link. From what I have seen neither ubuntu nor alpine do have a python to python3 link. At least the versions I checked.

Reply:
Why do we need to use python3 everwhere? 

For example, we know that Fedora uses python3 by default:
```
[avagin@laptop ~]$ env python -V
Python 3.7.6
[avagin@laptop ~]$ cat /etc/fedora-release 
Fedora release 31 (Thirty One)
```

I think it is enough if one of jobs will run on python3, is it?

Reply:
> I think it is enough if one of jobs will run on python3, is it?

I think it is more important to run as much on Python 3 and only use Python2 for old distributions like CentOS 7 and Ubuntu 16.04.

Reply:
I think Adrian refers to [PEP 394 -- The "python" Command on Unix-Like Systems](https://www.python.org/dev/peps/pep-0394/).
An alternative solution could be:
```
python3 -m venv virtualenv
source virtualenv/bin/activate
```